libs/components/packages/src/schematics/migrations/update-12/help-inline-import-path/help-inline-import-path.schematic.ts

```diff
@@ -26,7 +20,7 @@ export default function () {
         return;
       }
 
-      moveClassToLibrary(tree, path, sourceFile, content, {
```

Comment:
I needed to make some adjustments to the `moveClassToLibrary` function so that it could be called multiple times for a given file.